commit 1d9959734a1949ea4f2427bd2d8b21ede6b2441c
Author: Eric Paris <eparis@redhat.com>
Date:   Fri Aug 7 14:53:57 2009 -0400

    security: define round_hint_to_min in !CONFIG_SECURITY
    
    Fix the header files to define round_hint_to_min() and to define
    mmap_min_addr_handler() in the !CONFIG_SECURITY case.
    
    Built and tested with !CONFIG_SECURITY
    
    Signed-off-by: Eric Paris <eparis@redhat.com>
    Signed-off-by: James Morris <jmorris@namei.org>

Adjusted to apply to Debian's 2.6.24 by dann frazier <dannf@debian.org>

diff -urpN linux-source-2.6.24.orig/include/linux/security.h linux-source-2.6.24/include/linux/security.h
--- linux-source-2.6.24.orig/include/linux/security.h	2010-01-31 17:55:54.000000000 -0700
+++ linux-source-2.6.24/include/linux/security.h	2010-01-31 17:56:40.000000000 -0700
@@ -117,8 +117,6 @@ struct request_sock;
 #define LSM_UNSAFE_PTRACE	2
 #define LSM_UNSAFE_PTRACE_CAP	4
 
-#ifdef CONFIG_SECURITY
-
 /*
  * If a hint addr is less than mmap_min_addr change hint to be as
  * low as possible but still greater than mmap_min_addr
@@ -131,9 +129,11 @@ static inline unsigned long round_hint_t
 		return PAGE_ALIGN(mmap_min_addr);
 	return hint;
 }
-
 extern int mmap_min_addr_handler(struct ctl_table *table, int write, struct file *filp,
 				 void __user *buffer, size_t *lenp, loff_t *ppos);
+
+#ifdef CONFIG_SECURITY
+
 /**
  * struct security_operations - main security structure
  *
